tsdb/index/index_test.go

```diff
@@ -505,3 +506,48 @@ func TestNewFileReaderErrorNoOpenFiles(t *testing.T) {
 	// dir.Close will fail on Win if idxName fd is not closed on error path.
 	dir.Close()
 }
+
+func TestSymbols(t *testing.T) {
+	buf := encoding.Encbuf{}
+
+	buf.PutUvarintStr("something")
+
+	symbolsStart := buf.Len()
+	buf.PutBE32int(204)
+	buf.PutBE32int(100)
+	for i := 0; i < 100; i++ {
+		buf.PutUvarintStr(string(i))
```

Comment:
These symbols will not be in order, you'd need fixed size strings.

Reply:
Or add your test to TestPostingsMany, which already has all the required infrastructure.

Reply:
I think a separate unit test for separate symbols structure would be nice (:

Additionally, those symbols are in perfect order, when you print `string(i)`:   `       	 
                     ! " # $ % & ' ( ) * + , - . / 0 1 2 3 4 5 6 7 8 9 : ; < = > ? @ A B C D E F G H I J K L M N O P Q R S T U V W X Y Z [ \ ] ^ _ ` a b c `

They would be not in order if I would do `fmt.Sprint(i)` (:

Reply:
That needs a comment then, I'd intuitively expect it to be the decimal representation of the int

Reply:
done